cmd/manager/main.go

```diff
@@ -347,7 +346,7 @@ func execute() {
 		log.Error(err, "unable to create controller", "controller", "EnterpriseSearch")
 		os.Exit(1)
 	}
-	if err = associationctl.AddApmES(mgr, accessReviewer, params); err != nil {
+	if err = controller.AddApmES(mgr, accessReviewer, params); err != nil {
```

Comment:
I wonder if the `associationctl` alias was not better and we should instead remove the other import.